src/main/java/com/google/sps/data/HabitList.java

```diff
@@ -1,16 +1,19 @@
 
-package main.java.com.google.sps.data;
+package com.google.sps.data;
+
+import com.google.cloud.datastore.TimestampValue;
 
 public final class HabitList {
     
 
-    private final TimeStamp notifyTime;
-    private final string listName;
-    private final string username;
+    //private final TimestampValue notifyTime;
```

Comment:
If this is commented out, leave a reason why. 
e.g.
// TODO: Implement notifyTime later.